stable/mysql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: mysql
-version: 1.2.1
+version: 1.2.2
```

Comment:
Can you bump the minor version here as a new feature has been added

Reply:
@olemarkus , I have bump to version 1.2.3

Reply:
@olemarkus ,
may you please review?

Reply:
Hey. The minor version still needs to be bumped

Reply:
Thanks @olemarkus ,
Version bumped to 1.3.0